flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/enumerator/KafkaEnumeratorTest.java

```diff
@@ -300,7 +300,7 @@ public void testDiscoverPartitionsPeriodically() throws Throwable {
                     getAllAssignSplits(context, PRE_EXISTING_TOPICS);
             assertThat(initialPartitionAssign)
                     .extracting(KafkaPartitionSplit::getStartingOffset)
-                    .containsOnly(KafkaPartitionSplit.LATEST_OFFSET);
+                    .containsOnly((long) KafkaSourceTestEnv.NUM_RECORDS_PER_PARTITION);
```

Comment:
nit: redundant cast

Reply:
I had been double check, removing it will results in a syntax error.